app/src/organisms/Devices/ProtocolRun/ProtocolRunSetup.tsx

```diff
@@ -254,25 +262,42 @@ export function ProtocolRunSetup({
 
 interface StepRightElementProps {
   stepKey: StepKey
-  calibrationStatus: ProtocolCalibrationStatus
+  calibrationStatusRobot: ProtocolCalibrationStatus
+  calibrationStatusModules?: ProtocolCalibrationStatus
   runHasStarted: boolean
+  isFlex: boolean
```

Comment:
nit, props that can be undefined should be listed last

```suggestion
  calibrationStatusRobot: ProtocolCalibrationStatus
  runHasStarted: boolean
  isFlex: boolean
  calibrationStatusModules?: ProtocolCalibrationStatus
```